app.py

```diff
@@ -0,0 +1,429 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import pickle
+import seaborn as sns
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.model_selection import RandomizedSearchCV, GridSearchCV
+from sklearn.metrics import r2_score
+from sklearn.metrics import mean_squared_error
+from sklearn.metrics import mean_absolute_error
+from mpl_toolkits.mplot3d import Axes3D
+from PIL import Image
+from plotly.offline import iplot
+import plotly as py
+import plotly.tools as tls
+import cufflinks as cf
+import plotly.figure_factory as ff
+import plotly.express as px
+import streamlit.components.v1 as components
+
+
+py.offline.init_notebook_mode(connected = True)
+cf.go_offline()
+cf.set_config_file(theme='white')
+
+df = pd.read_csv("forestfires.csv")
+#all functions
+# @st.cache
+def welcome():
+
+    return "Welcome All"
+# @st.cache    
+def predict_forest_fire(x,y,month,ffmc,dmc,dc,temp):
+    prediction = model.predict([[x,y,month,ffmc,dmc,dc,temp]])
+    print(prediction)
+    return prediction
+
+# @st.cache
+def main_prediction():
+    # st.title("Forest Fire Prediction")
+    # st.markdown("""<center><img src='https://upload.wikimedia.org/wikipedia/commons/d/d8/Deerfire_high_res_edit.jpg' width='500px'></center>""",unsafe_allow_html=True)
+    # st.markdown("""<br>""",unsafe_allow_html=True)
+    
+    html_temp = """
+    <div style="background-color:#00b3ff;padding:10px">
+    <h2 style="color:white;text-align:center;">Forest Fire Prediction</h2>
+    </div>
+    """
+    st.markdown(html_temp,unsafe_allow_html=True)
+    st.markdown("""<br>""",unsafe_allow_html=True)
+    x = st.text_input("X")
+    y = st.text_input("Y")
+    month = st.text_input("Month")
+    ffmc = st.text_input("FFMC")
+    dmc = st.text_input("DMC")
+    dc = st.text_input("DC")
+    temp = st.text_input("Temp")
```

Comment:
Add some placeholder texts to let the user know what kind of values the model is expecting. Take information from [here](https://github.com/dsc-iem/AI-Hacktoberfest/blob/master/README.md).

Also it will be great to have some field hints or info explaining what these fields mean.

Reply:
@soumya997 Also make sure to add the deployed link in README

Reply:
in the st.text_input() I have added some placeholder text like this 👇, and also edited the readme with the deployed link.
<be>
<img src="https://i.imgur.com/Y2dyz9d.png">
I hope it will do the job.